lib/v2/bangumi/tv/subject/index.js

```diff
@@ -1,26 +1,28 @@
 const getComments = require('./comments.js');
 const getFromAPI = require('./offcial-subject-api.js');
 const getEps = require('./ep.js');
+const { queryToBoolean } = require('@/utils/readable-social');
 
 module.exports = async (ctx) => {
     const id = ctx.params.id;
+    const type = ctx.params.type || 'ep';
+    const showOriginalName = queryToBoolean(ctx.params.showOriginalName);
     let response;
-    if (ctx.params.type) {
-        switch (ctx.params.type) {
-            case 'comments':
-                response = await getComments(id, Number(ctx.query.minLength) || 0);
-                break;
-            case 'blogs':
-                response = await getFromAPI('blog')(id);
-                break;
-            case 'topics':
-                response = await getFromAPI('topic')(id);
-                break;
-            default:
-                throw Error(`暂不支持对${ctx.params.type}的订阅`);
-        }
-    } else {
-        response = await getEps(id);
+    switch (type) {
+        case 'ep':
+            response = await getEps(id, showOriginalName);
+            break;
+        case 'comments':
+            response = await getComments(id, Number(ctx.query.minLength) || 0);
+            break;
+        case 'blogs':
+            response = await getFromAPI('blog')(id, showOriginalName);
+            break;
+        case 'topics':
+            response = await getFromAPI('topic')(id, showOriginalName);
+            break;
+        default:
+            throw Error(`暂不支持对${ctx.params.type}的订阅`);
```

Comment:
```suggestion
            throw Error(`暂不支持对${type}的订阅`);
```
https://github.com/DIYgod/RSSHub/blob/628c5cf84a43cb6b84dbed325ff18f8e802b2a31/lib/v2/bangumi/tv/subject/index.js#L8